src/Seo/Metadata.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types=1);
+
+namespace Wizaplace\SDK\Seo;
+
+final class Metadata
+{
+    /** @var null|string */
+    private $metaTitle;
```

Comment:
```suggestion
    private $title;
```
Vu que l'on est déjà dans la classe Meta, il n'est pas forcément nécessaire de préfixé de meta les variables.